controls/datagrid/columns/overview.md

```diff
@@ -1,77 +1,49 @@
-  * Auto&mdash;The columns is sized to its desired width. That is the maximum desired width of all associated cells.
-* `Width`(`double`)&mdash;Specifies the fixed width for the column. Applicable when the `SizeMode` property is set to `DataGridColumnSizeMode.Fixed`.
-* `MinimumWidth`(`double`)&mdash;Specifies the minimum width of a column. This property is applicable when setting `SizeMode` column property to `Fixed`. When `Minimumwidth` is set, you can not reduce the width of the column to a value lower than the `MinimumWidth`. 
-* `ActualWidth`(`double`)&mdash;Gets the actual width of the column.
-* `IsResizable`(`bool`)&mdash;Specifies whether the user can resize the DataGrid Column. The default value is `True`. This is only supported in `WinUI` and `MacCatalyst`.
-* `IsAutoGenerated`(bool)&mdash;Gets a value indication whether the column is auto-generated internally.
-* `CanUserEdit`(`bool`)&mdash;Specifies whether the user can edit the values in this column.
-* `CanUserFilter`(`bool`)&mdash;Specifies whether the user can filter this column by using the built-in Filtering UI.
-* `CanUserSort`(`bool`)&mdash;Specifies whether the user can sort the data by the values in this column.
-* `IsVisible`(`bool`)&mdash;Gets a value indicating if a specific column should be visualized.
-* `CellDecorationStyle`(`DataGridBorderStyle`)&mdash;Defines the Style object that defines the background of each cell associated with this column.
-* `CellDecorationStyleSelector`(DataGridStyleSelector): Defines the `StyleSelector` instance that allows for dynamic decoration on a per cell basis.
-* `CellContentTemplate`(`DataTemplate`)&mdash;Defines the appearance of each cell associated with concrete column.
-* `CellEditTemplate`(`DataTemplate`)&mdash;Defines the editor associated with the concrete column. The `CellEditTemplate` is displayed when the cell is in edit mode.
-* `IsFrozen`(`bool`)&mdash;Specifies whether the column is frozen. The default value is `False`.
-* `IsResizable`(`bool`)&mdash;Specifies whether the user can resize the DataGrid Column. The default value is `True`. This is only supported in `WinUI` and `MacCatalyst`.
+For the typed columns (Text, Numerical, Boolean, Date, Time and ComboBox) you can define which property of the underlying data object the column represents in the following ways:
 
->tip More information about `CellDecorationStyle` and  `CellDecorationStyleSelector` can be found in [Columns Styling]({%slug datagrid-columns-styling%}) topic.
+* `PropertyName`&mdash;Specifies the name of the property of the data object being displayed in the column's cells.
+* `DataMemberBinding`&mdash;Defines the binding which points to the data member of the underlying object being displayed in the column's cell. With `DataMemberBinding` you have control over the way data is formatted and displayed in the DataGrid cells, for example you can add a string formatter or a value converter.
```

Comment:
With `DataMemberBinding`, you have control over the way data is formatted and displayed in the DataGrid cells, for example, you can add a string formatter or a value converter.